src/js/components/RadioButton/StyledRadioButton.js

```diff
@@ -108,6 +109,10 @@ const StyledRadioButtonBox = styled.div`
     )};
   ${(props) => props.focus && focusStyle()};
```

Comment:
The solution you have is working but I'm thinking we may want to utilize the focus prop we have here on line 110 and get that working instead of adding a separate way to trigger focus styling.
To do this I think we just need to change `RadioButton.js` to set focus to true/false when `onFocus` or `onBlur` is triggered. You can take a look at the focus code in CheckBox.js to get an idea of how this is working it that component. Also I think we should default `focusIndicator` to true (we only want to turn off the focus indicator in special cases such as when the component is wrapped in a FormField). Again you can follow pretty much the same pattern CheckBox.js is for handling the focus stuff. Let me know if there are any questions!